CHANGELOG.md

```diff
@@ -206,8 +206,8 @@
 
 - New `command` input
   - This allows you to specify the Wrangler command you would like to run.
-    For example, if you want to publish the production version of your Worker you may run `publish --env=production`.
-  - This opens up other possibilities too like publishing a Pages project: `pages publish <directory> --project-name=<name>`.
```

Comment:
We should update command to `deploy` as well https://developers.cloudflare.com/workers/wrangler/commands/#pages

Reply:
I saw this, but assumed we wouldn't want to retroactively update a changelog since this is tied to a specific version.

Reply:
I meant to remove this completely